odk1-src/launch-apps-from-collect.rst

```diff
@@ -21,7 +21,21 @@ Launching External Apps
 Launching external apps to populate single fields
 ------------------------------------------------------
 
-ODK Collect can launch external applications to populate string, integer or decimal fields using the ``ex:intentString`` appearance. A ``value`` parameter that holds the current value for that field is passed to the application. Since v1.4.3, additional parameters can be specified. The names of these parameters are user defined and ``value`` is the only reserved name. 
+ODK Collect can launch external applications to populate string, integer or decimal fields using the ``ex:intentAction(intentExtras)`` appearance.
+
+Under the covers, this feature configures an intent, a messaging objects used to request an action from another app component. For more details on intents,  refer to `these Android docs <https://developer.android.com/guide/components/intents-filters.html>`_.
+
+Collect uses the text immediately after ``ex:`` and before an opening parenthesis if it exists as the action the intent is built with. This action must be namespaced. That is, ``ex:org.opendatakit.counter`` is valid but ``ex:counter`` is not.
+
+The parameters defined in the optional parentheses represent extended data ("extras") to be added to the intent. The value to the left of the equals sign represents the extra name and may require a namespace. The value to the right of the equals sign represents the extra value. Multiple extras can be configured using a comma-separated list. The extra values can be:
+
+  - XPath expressions referring to other fields and including function calls
+  - String literals defined in single quotes
+  - Raw integers or decimals
+
+An extra named ``value`` that holds the current value for the current field is always passed with the intent. Since v1.4.3, additional parameters with user-defined names can be specified. There are two reserved names: ``value`` and ``uri_data``.
+
```

Comment:
Any reason these two references (above and below this comment) to versions aren't using the [versionadded](http://www.sphinx-doc.org/en/stable/markup/para.html#directive-versionadded) or [versionchanged](http://www.sphinx-doc.org/en/stable/markup/para.html#directive-versionchanged) directives?

(It's okay if the reason is that we still haven't styled versionadded (#539) and we just don't like how it looks...)